OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -631,6 +670,94 @@ void AssignRolesToIdleUnits(Actor self)
 			}
 
 			FindAndDeployBackupMcv(self);
+
+			if (--minCaptureDelayTicks <= 0)
+			{
+				minCaptureDelayTicks = Info.MinimumCaptureDelay;
+				QueueCaptureOrders();
+			}
```

Comment:
Just noticed a long-standing bug here, a bit higher up we do:
```c#
if (--assignRolesTicks > 0)
    return;
```
which means we will only decrement `minAttackForceDelayTicks` and now `minCaptureDelayTicks` much less frequently than we expect!

We should address this either here, or in another PR that you'll need to rebase onto.